docs/howto.md

```diff
@@ -188,6 +188,94 @@ future. Do note that by separating the credentials in the secret, the password
 is never directly accessed by the operator itself.
 
 
+# Use a cluster internal share as persistent volume (without auto-provisioning)
+
+Kubernetes does not by default support mounting `smb` shares.  For now
+it's neccessary to install the [smb csi
+driver](https://github.com/kubernetes-csi/csi-driver-smb).  Please refer
+to its documentation on how to install the csi driver.
+
+First create a (non provisioning) storage class to differentiate smb shares
+from other storage.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: smb
+parameters:
+  type: smb
+provisioner: kubernetes.io/no-provisioner
+reclaimPolicy: Retain
+volumeBindingMode: Immediate
+```
+
+Then create a secret containing the username and password to mount the share.
+These are AD credentials when using AD authentication or the username and
+passwords as configured above.
```

Comment:
Another Nit - "as configured below"

Reply:
But aren't they configured above, in the section "Configure a share with custom users" or the section after?

Reply:
true. Sorry.